### DISCUSSION TITLE
Navbar is not working?

### DISCUSSION THREAD

**Discussion by razain** [gregsullivan/_tw#91] [General]
Hi everyone, I manage to setup the theme on WordPress but when I activate the theme the navbar seems not working, any idea?

![image](https://github.com/gregsullivan/_tw/assets/152825318/02a5a5a6-b61d-40db-a12c-51294d9f5672)

**Comment by gregsullivan** [comment]
Hello—Thanks for posting this!

Have you done any work to style the navigation? In its initial state, no styles have been added to the navigation. If you look in `./theme/template-parts/layout/header-content.php`, you'll see this:

https://github.com/gregsullivan/_tw/blob/e2a9eba2d24c1946fce180403720ffd25d3b9ee2/theme/template-parts/layout/header-content.php#L36-L44

You could style the navigation by adding `menu_class` and `container_class` parameters to the [`wp_nav_menu`](https://developer.wordpress.org/reference/functions/wp_nav_menu/) call.

Please let me know if you have any other questions!
